packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/android/processresponse.md

```diff
@@ -0,0 +1,49 @@
+The following code shows these steps in order:
+
+```kotlin
+private suspend fun handleResponse(response: IdxResponse) {
+    // Check if sign-in is successful.
+    if (response.isLoginSuccessful) {
+        // Get the access token and exit the flow.
```

Comment:
```suggestion
        // Get the OIDC token and exit the flow.
```

Reply:
See comment for "gettingatoken.md" line 5. The resolution for this will follow the resolution for that one.

Reply:
AFAIK we call it an access token in the docs. I'm leaving that.